### ISSUE TITLE
CSLR dataset request

### ISSUE THREAD

**Issue by YangLeiSX** [cslr-lipreading/CSLR#3]
Hello,
I am reading your paper titled "Language Without Borders: A Dataset and Benchmark for Code-Switching Lip Reading".
I noticed that you mentioned the CSLR dataset. However, I couldn't find a link to the dataset in the repository. 

Could you please provide the download link or any relevant information regarding how to access the dataset?
Thank you for your assistance, and I appreciate the effort you put into this work!

**Comment by akshatsanghvi211103** [comment]
Yeah it would be great if you could release the dataset @cslr-lipreading  !
